modules/@apostrophecms/ui/ui/apos/lib/i18next.js

```diff
@@ -107,5 +107,15 @@ export default {
         return result;
       }
     };
+
+    function canonicalize(locale) {
+      const [ language, territory ] = locale.split('-');
+      if (territory) {
+        return `${language}-${territory.toUpperCase()}`;
```

Comment:
Not a blocker but you might lowercase the language in order to respect the convention i18next is using (because they seem picky about the locale format).